bin/workflows/pycbc_create_offline_search_workflow

```diff
@@ -571,15 +561,15 @@ for inj_file, tag in zip(inj_files, inj_tags):
     #foundmissed/sensitivity plots
     s = wf.make_sensitivity_plot(workflow, found_inj, sensdir,
                                  exclude=['all', 'summ'], require='sub',
-                                 tags=ctags)
+                                 tags=[tag])
     f = wf.make_foundmissed_plot(workflow, found_inj, injdir,
                                  exclude=['all', 'summ'], require='sub',
                                  tags=[tag])
 
     # FIXME: It's not clear to me that this does anything??
     # Extra foundmissed/sensitivity plots
```

Comment:
to clarify this 'FIXME' (ie make it possible to remove that line) I suggest the following extra comment .. 
'This will make more injection plots for any plot_sensitivity or plot_foundmissed sections in the config file that do not have 'all', 'summ' or 'sub' in their tag names'

Reply:
I see some discussion of this in the SLACK and above, and apologies for my lack of clarity here.  

I guess I would be in favour of just removing this. These plots are already in sub-pages and do not need to be hidden further.

Reply:
OK, then it's simpler just to remove both the comments and the 'extra' calls to sensitivity/foundmissed plots, as you and Alex both think so.

Reply:
Done